e2e/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "set-e2e",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "lint": "eslint 'tests/**/*.ts{,x}'",
+    "playwright-local": "BASE_URL=http://localhost:8080 playwright test",
+    "playwright-local:win": "set BASE_URL=http://localhost:8080& playwright test",
+    "playwright-local-ui": "BASE_URL=http://localhost:8080 playwright test --ui",
+    "playwright-local-ui:win": "set BASE_URL=http://localhost:8080& playwright test --ui",
+    "playwright-local-debug": "BASE_URL=http://localhost:8080 playwright test --debug",
+    "playwright-local-debug:win": "set BASE_URL=http://localhost:8080& playwright test --debug",
```

Comment:
I don't think we need to set base url for local as it is set by process.env by default or "http://localhost:8080" so we shouldn't need to add it in the local scripts and we shouldn't need the win scripts for local.